main/src/ca/uwaterloo/flix/language/phase/jvm2/GenExpr.scala

```diff
@@ -0,0 +1,78 @@
+package ca.uwaterloo.flix.language.phase.jvm2
+
+import ca.uwaterloo.flix.language.ast.JvmAst
+import java.lang.classfile.CodeBuilder
+
+object GenExpr {
+  // signature (needs java24)
```

Comment:
@magnus-madsen the plan is to have this PR be a full vertical prototype right? So Cade should just upgrade to java 24 on his branch?